charts/server/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: server
 description: A Helm chart for the Woodpecker server
 type: application
-version: 0.2.0
+version: 0.3.0
```

Comment:
@genofire If you want to bump the subchart, this should be 1.0.0 due to the breaking changes.